ts/screens/profile/PrivacyMainScreen.tsx

```diff
@@ -58,96 +43,121 @@ const getRequestProcessingAlertSubtitle = () => ({
   DELETE: I18n.t("profile.main.privacy.removeAccount.alert.oldRequestSubtitle")
 });
 
-class PrivacyMainScreen extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = { requestProcess: false };
-  }
+/**
+ * A screen to show the main screen of the Privacy section.
+ * Here the user can:
+ * - display the accepted privacy policy (or Term of Service)
+ * - send a request to delete his profile
+ * - send a request to export all his data
+ */
+const PrivacyMainScreen = ({ navigation }: Props) => {
+  const dispatch = useIODispatch();
+  const userDataProcessing = useSelector(userDataProcessingSelector);
+  const prevUserDataProcessing = usePrevious(userDataProcessing);
+  const [requestProcess, setRequestProcess] = useState(false);
 
-  public componentDidMount() {
+  // the state returns to pot.none to every get - we want to see the loader only during the load of the requests,
+  // not when the request is confirmed by the user
+  const isLoading =
+    (pot.isNone(userDataProcessing.DELETE) &&
```

Comment:
Why are we excluding SomeLoading?

Reply:
We are excluding SomeLoading because during the request we update the state with `pot.toLoading(pot.none)`